Buildings/package.mo

```diff
@@ -460,23 +460,38 @@ have been <b style=\"color:blue\">improved</b> in a
                        This is for <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2124\">#2124</a>.<br/>
                        For Dymola, a conversion script makes this change.</td>
 </tr>
+<tr><td valign=\"top\">Buildings.Controls.OBC.CDL.Continuous.Sources.Pulse
+    </td>
+    <td valign=\"top\">Renamed parameter <code>startTime</code> to <code>delay</code>.<br/>
+                       This is for <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2170\">#2170</a>.<br/>
+                       For Dymola, a conversion script makes this change.
```

Comment:
Do we have a pointer to the location of Dymola conversion scripts somewhere at the beginning of these revision notes?

Reply:
A pointer is not needed because the conversion script is automatically involved per the Modelica Language Definition.